packages/valory/skills/decision_maker_abci/policy.py

```diff
@@ -131,9 +131,8 @@ def best_tool(self) -> str:
     def update_weighted_accuracy(self) -> None:
         """Update the weighted accuracy for each tool."""
         self.weighted_accuracy = {
-            tool: (acc_info.accuracy / 100)
-            * (acc_info.requests - acc_info.pending)
-            / self.n_requests
+            tool: acc_info.accuracy
```

Comment:
You forgot to add the "/ 100" at the acc_info.accuracy

Reply:
Did you see https://github.com/valory-xyz/trader/pull/296#discussion_r1681337009 above?